### PR TITLE
Please examine, will squash later blinkr fixes / testing

### PR DESCRIPTION
**Pull request by LightGuard**

### CODE REVIEW

File: _layouts/base.html.slim [hunk 1]
```diff
@@ -203,28 +203,28 @@ body(class="#{page.url.gsub('/','').split('.').first || 'home'} #{page.body_clas
...
-    script src="#{site.base_url}/#{site.context_url}/javascripts/terms-and-conditions.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/search.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/os-filter.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/social.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/visual.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/microsite.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/connectors.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/experiments.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/overlay.js"
-    script src="#{site.base_url}/#{site.context_url}/javascripts/devnation2015.js"
```
Comment (paulrobinson) [inline _layouts/base.html.slim]: I think 'site.context_url' is needed for staging builds.

File: _layouts/solution-base.html.slim [hunk 2]
```diff
@@ -5,4 +5,4 @@ layout: technology-base
 =content
 
 = javascripts("solutions", true) do
-  script src="#{site.base_url}/#{site.context_url}/javascripts/vendor/chosen.jquery.min.js"
```
Comment (paulrobinson) [inline _layouts/solution-base.html.slim]: More context_url issues

File: _layouts/solution.html.slim [hunk 3]
```diff
@@ -21,4 +21,4 @@ layout: technology-base
     =content
 
 = javascripts("solutions", true) do
-  script src="#{site.base_url}/#{site.context_url}/javascripts/vendor/chosen.jquery.min.js"
```
Comment (paulrobinson) [inline _layouts/solution.html.slim]: More context_url issues

File: forums.html.slim [hunk 4]
```diff
@@ -42,4 +42,4 @@ description: Forums are the primary way that Red Hat Developers communicate.
                 = product.forum.histogram.to_json
 = javascripts("forums", true) do
   script src="https://www.google.com/jsapi?autoload=#{URI.escape({ 'modules' => [ { 'name' => 'visualization', 'version' => '1.0', 'packages' => [ 'corechart' ] } ] }.to_json)}"
-  script src="#{site.base_url}/#{site.context_url}/javascripts/forums.js"
```
Comment (paulrobinson) [inline forums.html.slim]: more context_url issues

**Comment by paulrobinson** [comment]
Just deleted my comments about the 'context_url'. 

Is the reason why you removed the usage of 'context_url' because it was always empty? Is this the case for staging builds too?

**Comment by paulrobinson** [comment]
Otherwise this looks fine.

**Comment by LightGuard** [comment]
The context_url isn't used and its adding warnings to the blinkr report.

On Wednesday, August 26, 2015, Paul Robinson <notifications@github.com>
wrote:

> Otherwise this looks fine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/359#issuecomment-134935836>
> .
>


-- 
Sent from Gmail Mobile

**Comment by LightGuard** [comment]
I'd like to keep this open for a little bit (maybe?) and continue to work on fixes, but if we feel like this is good for now we can continue work in a different branch.

**Comment by paulrobinson** [comment]
Let me know when this is ready for review.

**Comment by LightGuard** [comment]
Yes, this is ready for review, and it looks like I need to merge with HEAD again :(

**Comment by paulrobinson** [comment]
Yep, can you squash whilst you're at it too please?

**Comment by LightGuard** [comment]
I don't feel good squashing this. I looked through it and all of the
current commits are as small as I feel they should be.

On Wed, Sep 2, 2015 at 3:25 AM, Paul Robinson <notifications@github.com>
wrote:

> Yep, can you squash whilst you're at it too please?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/359#issuecomment-136995288>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Pull request merged**